### ISSUE TITLE
Please add an option to set .stl as project thumbnail

### ISSUE THREAD

**Issue by gitatmax** [getdepot/depot#1]

**Comment by destari** [comment]
So just basically allow the ability to skip the step where you make a snapshot, and just go straight to the thumbnail?

**Comment by gitatmax** [comment]
@destari Yeah, that's a simpler way to put it! Exactly. Basically to skip that initial step and to pretty much always have a preview.
